src/Plugin.php

```diff
@@ -44,7 +44,12 @@ public static function plugins_loaded() {
   public static function init() {
     add_action('woocommerce_after_single_product', __NAMESPACE__ . '\TrustedShops::woocommerce_after_single_product');
     add_filter('woocommerce_thankyou_order_received_text', __NAMESPACE__ . '\TrustedShops::woocommerce_thankyou_order_received_text', 100, 2);
-    add_action('woocommerce_order_items_table', __NAMESPACE__ . '\TrustedShops::woocommerce_order_items_table');
+    if (WC_VERSION < '3.3.0') {
```

Comment:
@LucaPipolo @juanlopez4691 In situations like this, just replace the deprecated function with the new one.

If this plugin happens to be used in a project that hasn't the newer version of WooCommerce yet, then we simply expect that project to update woocommerce, too, instead of adding permanent code debt to our project.

Reply:
I don't agree with this.

There are two kind of situations in which we'll troubles with this:
1. if we install `woocommerce-reputations` plugin on a new project where WC is < then 3.3.0
2. if we need to update `woocommerce-reputations` plugin separately, e.g. because of an hotfix or of a new feature


IMHO, on both cases, we should not depends on the generally long process of "updates and QAs everything". 

This is why, after consider it, I choose the current approach of retro-compatibility. As almost every other woocommerce related plugins do.

Reply:
None of our plugins cares for backwards-compatibility currently and we did not have a problem with that in the past years.

If you add backwards-compatible code, then you normally also need to add the whole infrastructure around it:

1. At minimum an inline comment that states since when the new API exists as well as a clearly defined date/event at which the old legacy code should be removed.

2. You would also have to test the old code path with an old version of WooCommerce to ensure that it actually still works (and not only with the new/current version), which should therefore be noted in the plugin's docs about testing.